CHANGES.md

```diff
@@ -1,3 +1,7 @@
+Version 2018.04.23
```

Comment:
It's 2019 already! Time flies!

Also, I wonder whether we shouldn't just switch to a naming scheme `year.release_number`, i.e., first release in 2019 would be `2019.1`, second release would be `2019.2` etc, WDYT?

Reply:
I'm all for it!